bolt-design-system/bolt
# Auto-add Bolt Dependencies

## Jira
N/A

## Summary
Updates Bolt Build Tools to auto-add Bolt dependencies missing from the consumer’s `.boltrc` config file.

## Details
- Updates Bolt Build Tools to auto-add dependencies specified in the component's package.json file. 
- Adds snapshot component rendering testing for the `<bolt-image>` and `<bolt-video>` to verify `<bolt-ratio>` dependency specified is getting pulled in and compiled as expected.
- Relocates the `<bolt-ratio>` object to the components folder to help confirm auto-resolved and added deps are working as expected.
- Remaps existing `@bolt/global` references to the previous bolt ratio object to point at the new `@bolt/components-ratio` component location to avoid introducing a breaking change here.

## How to test
- Confirm `<bolt-image>` and `<bolt-video>` demos continue to work as expected
- Confirm Jest tests added render as expected
